### ISSUE TITLE
Child routes not working inside RadSideDrawer (Cannot find primary outlet)

### ISSUE THREAD

**Issue by lazar-vuckovic** [telerik/nativescript-ui-samples-angular#13]
While trying to make a layout that has child views, I ran into the following issue:

```
CONSOLE LOG file:///app/tns_modules/@angular/core/src/facade/lang.js:345:16: EXCEPTION: Error: Uncaught (in promise): Error: Cannot find primary outlet to load 'ListViewComponent'

CONSOLE LOG file:///app/tns_modules/@angular/core/src/facade/lang.js:345:16: STACKTRACE:

CONSOLE LOG file:///app/tns_modules/@angular/core/src/facade/lang.js:345:16: resolvePromise@file:///app/tns_modules/zone.js/dist/zone-node.js:496:41
	file:///app/tns_modules/zone.js/dist/zone-node.js:473:28
	invoke@file:///app/tns_modules/zone.js/dist/zone-node.js:281:34
	onInvoke@file:///app/tns_modules/@angular/core/src/zone/ng_zone_impl.js:46:47
	invoke@file:///app/tns_modules/zone.js/dist/zone-node.js:280:43
	run@file:///app/tns_modules/zone.js/dist/zone-node.js:174:50
	file:///app/tns_modules/zone.js/dist/zone-node.js:529:61
	invokeTask@file:///app/tns_modules/zone.js/dist/zone-node.js:314:43
	onInvokeTask@file:///app/tns_modules/@angular/core/src/zone/ng_zone_impl.js:37:51
	invokeTask@file:///app/tns_modules/zone.js/dist/zone-node.js:313:55
	runTask@file:///app/tns_modules/zone.js/dist/zone-node.js:214:58
	drainMicroTaskQueue@file:///app/tns_modules/zone.js/dist/zone-node.js:432:43
	promiseReactionJob@[native code]
	UIApplicationMain@[native code]
	start@file:///app/tns_modules/application/application.js:233:26
	file:///app/tns_modules/nativescript-angular/application.js:153:26
	ZoneAwarePromise@file:///app/tns_modules/zone.js/dist/zone-node.js:542:38
	nativeScriptBootstrap@file:///app/tns_modules/nativescript-angular/application.js:151:23
	anonymous@file:///app/main.js:21:36
	evaluate@[native code]
	moduleEvaluation@[native code]
	[native code]
	promiseReactionJob@[native code]
```

The sidedrawer xml looks like this:

```
<RadSideDrawer #drawer>

    <template drawerSide>

        ...

    </template>

    <template drawerMain>

        <StackLayout>

            <tabs [items]="tabs" (tabItemTapped)="tabItemTapped($event)"></tabs>

            <router-outlet></router-outlet>

        </StackLayout>

    </template>

</RadSideDrawer>
```

with `tabs` being a custom component that changes the route on tap and the routes are set up like this:

```
const routes: RouterConfig = [

	...

    {

        path: "around-me",

        component: AroundMePage,

        children: [

            { path: 'list-view', component: ListViewComponent },

            { path: 'map-view', component: MapViewComponent }

        ]

    }

];
```

If I remove the sidedrawer component and place the `router-outlet` element inside any layout, it renders normally. Also, the sidedrawer works normally on other pages, that don't have the `router-outlet`.

**Comment by nestor94** [comment]
Same issue here.

**Comment by ahmedelgendy** [comment]
The same error with me, Please any solutions?

**Comment by Feather-CMS** [comment]
+1

**Comment by VladimirAmiorkov** [comment]
Hi all,

We are currently investigating issues related to using the RadSideDrawer in {N} with angular 2. @lazar-vuckovic  could you share with use how exactly you are navigating to the `router-outlet`, some code snippets of the `tabItemTapped` handled would be great.

**Comment by lazar-vuckovic** [comment]
@VladimirAmiorkov `tabItemTapped` had just a `console.log()` and it was breaking on page render, if I recall correctly, since it's been nearly a month since I first posted the issue.

**Comment by VladimirAmiorkov** [comment]
@lazar-vuckovic We managed to reproduce the issue and are currently investigating its possible solutions, thank you for bringing this to our attention.

**Comment by lazar-vuckovic** [comment]
@VladimirAmiorkov glad to be of help and looking forward to the solution. Thanks

**Comment by VladimirAmiorkov** [comment]
Resolved with version **1.3.1** of [nativescript-telerik-ui](https://www.npmjs.com/package/nativescript-telerik-ui)
